index.js

```diff
@@ -485,8 +485,8 @@
   //. The Maybe type represents optional values: a value of type `Maybe a` is
   //. either a Just whose value is of type `a` or a Nothing (with no value).
   //.
-  //. The Maybe type satisfies the [Monoid][], [Monad][], [Foldable][], and
-  //. [Extend][] specifications.
+  //. The Maybe type satisfies the [Monoid][], [Monad][], [Foldable][],
+  //. [Traversable][] and [Extend][] specifications.
```

Comment:
Traversable implies Foldable, so we could remove Foldable from the list in the interest of brevity.